### PR TITLE
Fixed links. Removed references to the old blocks

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

File: common.blocks/i-bem/i-bem.ru.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ $(уникальный селектор).bem('b-link');
 Модификаторы нельзя устанавливать, напрямую меняя `CSS` класс на соответствующей `DOM`-ноде. Для
 корректной работы `js` все манипуляции с модификаторами должны производиться при помощи
 метода-хелпера `setMod()`. Также существуют методы `hasMod()`, `getMod`/`getMods()`, `toggleMod()`
-и `delMod()`. Сигнатуры этих методов доступны в [референсе по BEM](/blocks/i-bem/i-bem.jsdoc.wiki).
+и `delMod()`. Сигнатуры этих методов доступны в [референсе по BEM](http://ru.bem.info/libs/bem-core/v2.5.0/desktop/i-bem/jsdoc/).
```
Comment (tadatuta) [inline common.blocks/i-bem/i-bem.ru.md]: please use `https` for any link to bem.info.

Comment (tadatuta) [inline common.blocks/i-bem/i-bem.ru.md]: use `current` instead of specific version if it's possible

File: common.blocks/i-bem/i-bem.ru.md [hunk 2]
```diff
@@ -627,8 +627,6 @@ BEM.DOM.decl('b-form-checkbox', {
 Третий параметр, передаваемый в функцию декларации блока, -- это хеш статических
 методов блока.
 
-Примером блока, использующего статические методы, может служить [/blocks/b-flash/b-flash.wiki](/blocks/b-flash/b-flash.wiki).
```
Comment (tadatuta) [inline common.blocks/i-bem/i-bem.ru.md:630]: you may refer to any block with static methods. e.g. https://ru.bem.info/libs/bem-core/current/touch-pad/ua/jsdoc/

File: common.blocks/i-bem/i-bem.ru.md [hunk 3]
```diff
@@ -760,7 +758,7 @@ DOM-представление блока после инициализации:
 <a name="liveinit"></a>
 ## Инициализация по требованию (liveInit)
 
-Многим блокам (например, `b-link`, `b-dropdown`, `b-smart-help`) нет необходимости делать сразу же полную инициализацию.
+Многим блокам нет необходимости делать сразу же полную инициализацию.
```
Comment (tadatuta) [inline common.blocks/i-bem/i-bem.ru.md]: it'd be great to add some contemporary examples. e.g. `select`

File: common.blocks/i-bem/i-bem.ru.md [hunk 4]
```diff
@@ -837,12 +835,7 @@ BEM.DOM.decl('b-my-block', {
 
 
 Список методов поиска блоков и их сигнатуры можно посмотреть в
-[референсе по BEM.DOM](/blocks/i-bem/dom/i-bem__dom.jsdoc.wiki).
-
-Примерами блоков, использующих методы поиска других блоков, могут быть:
-[b-smart-help](/blocks/b-smart-help/b-smart-help.wiki),
-[b-screenshot](blocks/b-screenshot/b-screenshot.wiki) и
-[b-dropdowna](blocks/b-dropdowna/b-dropdowna.wiki).
+[референсе по BEM.DOM](http://ru.bem.info/libs/bem-core/v2.5.0/desktop/i-bem/jsdoc/).
```
Comment (tadatuta) [inline common.blocks/i-bem/i-bem.ru.md]: same thing. `v2.5.0` -> `current` and proper examples for findBlock*

Comment (tadatuta) [inline common.blocks/i-bem/i-bem.ru.md]: I think we should replace all occurrences of `BEM.DOM` with `i-bem__dom` as it's module name

**Comment by gela-d** [comment]
@tadatuta please, check my next commit

**Comment by tadatuta** [comment]
thanks!

**Pull request merged**